packages/sensors_plus/sensors_plus/example/integration_test/sensors_plus_test.dart

```diff
@@ -24,6 +24,10 @@ void main() {
     expect(await completer.future, isNotNull);
   });
 
+  // The acceleration value is +9.81, which corresponds to the acceleration of
+  // the device (0 m/s^2) minus the force of gravity (-9.81 m/s^2)
+  // When the device lies flat on a table.
+  // You can refer to this link for more information.
```

Comment:
"You can refer to this link for more information."

There is no link :)

Reply:
Terrible~
Let me add this link.